controllers/goto/tags.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
     like: route(['query.filter.tagname.like']),
     random: route(['query.filter.random']),
     relatedToMyTags: route(['query.filter.relatedToMyTags']),
-    relatedToTags: route(['query.filter.relatedToTags'])
+    relatedToTags: route(['query.filter.relatedToTags']),
+    popularCount: route(['query.sort'], 'popularCount')
```

Comment:
the name "popularCount" is confusing me or it is not intuitive for me, I would give "popularByUses", or even "popularityCount",  or something closer to other names. Maybe because other names are not shortened. It is very minor preference issue of course.